src/main/sensors/gyro.c

```diff
@@ -357,7 +357,39 @@ STATIC_UNIT_TESTED void performGyroCalibration(gyroDev_t *dev, gyroCalibration_t
     gyroCalibration->calibratingG--;
 }
 
-void gyroUpdate(void)
+#ifdef ASYNC_GYRO_PROCESSING
+STATIC_FASTRAM float imuAccumulatedRate[XYZ_AXIS_COUNT];
+STATIC_FASTRAM timeUs_t imuAccumulatedRateTimeUs;
+static void imuUpdateGyroscope(timeDelta_t gyroUpdateDeltaUs)
```

Comment:
`imuUpdateGyroscope` is not a proper name for this function - now it has nothing to do with IMU. Maybe `updateAccumulatedRates` is a better name?